CHANGELOG.unreleased.md

```diff
@@ -16,3 +16,5 @@ Section headings should be at level 3 (e.g. `### Added`).
 ### Changed
 
 - Boolean values for the `reinit` setting are deprecated; use "return_previous" and "finish_previous" instead (@timoffex in https://github.com/wandb/wandb/pull/9557)
+
+- Moved helper methods to check for server feature flags from the public API to the internal API (@ibindlish in https://github.com/wandb/wandb/pull/9561)
```

Comment:
nit: I think last i checked the changelog was more for user facing changes?

Reply:
oh thanks! will remove this then

Reply:
this probably doesn't need to be in the changelog it is totally internal code organization that shouldn't affect any user